__tests__/articles.test.js

```diff
@@ -0,0 +1,87 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/articles/:article_id", () => {
+  describe("GET", () => {
+    describe("Status 200", () => {
+      it("should return the correct article given a valid article_id", () => {
+        return request(app)
+          .get("/api/articles/1")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.article).toHaveProperty("author");
+            expect(body.article).toHaveProperty("title");
+          });
```

Comment:
If the test that follows on from this also asserts the response should have an _author_ and _title_ keys, then this test has become unnecessary